controllers/controllers.js

```diff
@@ -70,7 +71,29 @@ exports.getComments = (request, response, next) => {
             response.status(200).send({ comments });
         })
         .catch((err) => {
-            console.log(err);
             next(err);
         });
+};
+
+exports.postComment = (request, response, next) => {
+    const review_id = request.params.review_id;
+    let newComment = request.body;
+
+    if (newComment.username === undefined) {
+        response
+          .status(400)
+          .send({ msg: "Please provide a username." });
+    } else if (newComment.body === undefined) {
+        response
+          .status(400)
+          .send({ msg: "Please provide a body." });
+    } else {    
```

Comment:
I would do the validation of the post body inside the model so that your controller is only handling the response.